frontend/src/index.html

```diff
@@ -143,9 +143,53 @@
     <div class="flex flex-1 mx-auto w-full max-w-screen-xl sm:overflow-y-hidden">
       <div id="filter-panel" class="fixed z-20 top-0 open:flex hidden flex-col w-64 max-w-full h-full text-sm bg-white md:static md:z-auto md:flex md:w-1/5 md:max-w-sm">
         <div class="flex-none p-2 text-center text-teal-800 font-bold bg-submarine-500 uppercase md:hidden">Filter</div>
-        <div class="flex-grow p-2 pb-40 pt-0 overflow-x-hidden overflow-y-auto">
-          <div id="language-list"></div>
+        <div class="flex-none p-2 pb-40 pt-0 overflow-x-hidden overflow-y-auto">
```

Comment:
I'm not sure we need to make it `flex-none`, please consider whether it is the reason behind this.

Reply:
There was a typo, I will change it back to flex-grow.